docs/book/src/Good Practices

```diff
@@ -0,0 +1,24 @@
+# Good Practices
+
+## What is "Reconciliation" in Operators?
+
+The reconciliation is a continuos  loop that performs necessary actions on the current state to ensure that the Custom Resource reaches the desired state as specified by the user.
```

Comment:
```suggestion
```

Remove this one we would need to shape this text and link within k8s and controller-runtime we can do that in a follow up.